app/templates/main/manageServiceLearningFaculty.html

```diff
@@ -3,7 +3,12 @@
 {% block scripts %}
   {{super()}}
   <script type="module" src="/static/js/manageServiceLearningFaculty.js"></script>
-  <script type="text/javascript" src="https://cdn.datatables.net/v/dt/dt-1.11.3/r-2.2.9/datatables.min.js"></script>
+  <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.11.3/js/jquery.dataTables.js"></script>
+
+  {% block styles %}
+      {{super()}}
+      <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.11.3/css/jquery.dataTables.css">
+    {% endblock %}

```

Comment:
Your style block needs to be outside the script block

Reply:
fixed